Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM quay.io/packit/base:c9s
 
 ENV USER=packit \
-    HOME=/home/packit
+    HOME=/home/packit \
+    PYTHONPATH="/usr/local/lib/python3.9/site-packages"
```

Comment:
I would add a comment or/and a link to the issue.

Reply:
Done, also added the note under the base image to adjust it when changing.